lib/api/authentication/web_authentication.py

```diff
@@ -48,7 +48,7 @@ def authenticate(self, request):
                 scheme, token = request.headers['Authorization'].split()
                 if scheme != 'Bearer':
                     raise ValueError
-            except ValueError:
+            except (ValueError, jwt.exceptions.InvalidSignatureError, jwt.exceptions.DecodeError):
```

Comment:
Not sure where the jwt errors would come?

Reply:
One the line below ;) but yeah it's wrong